clang/runtime/dpct-rt/include/dpct/codepin/detail/json.hpp

```diff
@@ -237,7 +241,8 @@ inline bool object::contains(const std::string &str) {
 inline bool json_parser::parse_string(std::string &ret_str) {
   while (peek() != '"') {
     if (cur_p == end) {
-      error_exit("The string needs \" to wrap it. Please check the json "
+      error_exit("[JSON Parser]: Parse string needs \" to wrap it. Please "
```

Comment:
key value of a json need to be wrapped in \". Please check the json file format.